src/digital_identity_service/index.js

```diff
@@ -80,6 +81,47 @@ class DigitalIdentityService {
       throw err;
     }
   }
+
+  /**
+   *
+   * @param sessionId
+   * @returns {Promise<ShareQrCodeCreateResult>}
+   */
+  async createQrCode(sessionId) {
+    Validation.isString(sessionId, 'sessionId');
+
+    const payload = new Payload({});
+
+    const requestBuilder = new RequestBuilder()
+      .withBaseUrl(this.apiUrl)
+      .withHeader('X-Yoti-Auth-Id', this.sdkId)
+      .withPemString(this.pem)
+      .withEndpoint(`/v2/sessions/${sessionId}/qr-codes`)
+      .withQueryParam('appId', this.sdkId)
+      .withMethod('POST')
+      .withPayload(payload);
```

Comment:
For creating a QR code, I think there is an optional payload, should we allow the user to add them? @dhivya-pattabhiraman

Reply:
`{
  "transport": "",     // optional - default is 'QR_CODE'
  "displayMode": ""    // optional - default is 'INLINE'
}`

This one? I am not even sure what that does to be honest

Reply:
Ok, I have been investigating and had some conversations, we can ignore that bit.